src/locales/fr/datepicker.js

```diff
@@ -0,0 +1,3 @@
+const format = 'YYYY-MM-DD';
```

Comment:
As far as I can remember in France they use DD/MM/YYYY, right?

Reply:
This is a good point. We might consider supporting some regional setting in parallel of localization, because 'YYYY-MM-DD' is definitely the preferable date format for fr-CA (as for en-CA). But this might be diferrent for other regions such fr-FR or en-US.

Reply:
Yes, we could do that. I need to check how we can make this easily possible without duplicating all the phrases though.
For now you could also just store the files in a fr_CA folder and make it really specific for Canada (then the config should be aligned to that, of couse.)